library/src/service/BasisTheoryService.ts

```diff
@@ -13,20 +13,26 @@ export abstract class BasisTheoryService<
   public readonly client: AxiosInstance;
 
   public constructor(options: T) {
-    const { apiKey, baseURL } = options;
+    const { apiKey, baseURL, transformRequest, transformResponse } = options;
+
     this.client = axios.create({
       baseURL,
       headers: {
         [API_KEY_HEADER]: apiKey,
       },
-      transformRequest: ([] as AxiosTransformer[]).concat(
-        options.transformRequest || transformRequestSnakeCase,
-        axios.defaults.transformRequest as AxiosTransformer[]
-      ),
-      transformResponse: (axios.defaults
-        .transformResponse as AxiosTransformer[]).concat(
-        options.transformResponse || transformResponseCamelCase
-      ),
+      transformRequest: [
```

Comment:
We should rollback this block, because the new one doesn't support both instance/array of `AxiosTransformer` from `options.transformRequest` and `options.transformResponse`.